src/conditions/blockingTitle.ts

```diff
@@ -0,0 +1,20 @@
+import { ConditionConfig } from './../config'
+import { PullRequestInfo } from '../models'
+import { ConditionResult } from '../condition'
+
+export default function doesNotHaveBlockingTitle (
+  config: ConditionConfig,
+  pullRequestInfo: PullRequestInfo
+): ConditionResult {
+  const regexObj = new RegExp(config.blockingTitleRegex, 'i')
+
+  if (regexObj.test(pullRequestInfo.title)) {
+    return {
+      status: 'fail',
+      message: `Blocking words are found (${ pullRequestInfo.title })`
```

Comment:
`Blocking words are found in title` would suffice in this case. The title itself doesn't give the user much context in this regard.